src/style.css

```diff
@@ -0,0 +1,119 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+form {
+  outline: none;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+
+  --border-color: #e1e7ed;
```

Comment:
It is better to use abstract naming, like "primary-color", "secondary-color", because you may want to use this color for background or other rules, not only for border